### ISSUE TITLE
Add "Hi <ContactName>" to replies in Email Inbox

### ISSUE THREAD

**Issue by aslamabbas** [chatwoot/chatwoot#5053]
**Is your enhancement request related to a problem? Please describe.**
Typing "Hi Name" is a repetitive process during email replies.

**Describe the solution you'd like**
Automatically add "Hi <contactName>" to the start of the conversation.

**Describe alternatives you've considered**

**Comment by iammattmartin** [comment]
This would have to be selective as not everyone will be sending a message with Hi or a greeting such as this. I know for us, any follow-up messages have no greeting as they don't require one because it is not an email conversation.

**Comment by aslamabbas** [comment]
@iammattmartin Yes ,this may only makes sense in Email Inbox and my request is to add the same only in Email Inbox
